bpf/include/bpf/builtins.h

```diff
@@ -12,11 +12,24 @@
 # define lock_xadd(P, V)	((void) __sync_fetch_and_add((P), (V)))
 #endif
 
+/* Unfortunately verifier forces aligned stack access while other memory
```

Comment:
If ptr leaks are OK and the return type is not a pointer we could probably allow unaligned stack access, any idea if that would help performance?

Reply:
You mean for the memcpy case (not this PR)? BPF stack requires alignment, I think if we we'd try to get rid of it from kernel side then it might be at the cost of higher complexity perhaps. If we'd only keep spilled pointers aligned, it could work, agree. Though we won't be able to get rid of __align_stack_8 for older kernels.

Reply:
right memcpy. Otherwise makes sense to me.